README.md

```diff
@@ -37,17 +37,26 @@ swiftmailer:
         spool: { type: memory }
 ```
 
-3. Implement your custom configuration class. This should implement the `Configuration\ConfigurationInterface`. For 
+4. Implement your custom configuration class. This should implement the `Configuration\ConfigurationInterface`. For
 an implementation example, have a look at `Configuration\EmptyConfiguration`. Note that the custom file is required:
 it is checked whether the `EmptyConfiguration` is still loaded, which will make this bundle throw an exception if that
 is the case.
 
-4. Register your configuration class in the kernel parameters (either `parameters.yml` or `config.yml`):
+5. Register your configuration class in the kernel parameters (either `parameters.yml` or `config.yml`):
 ```yml
 parameters:
     kickin.exceptionhandler.configuration.class: <your_class_here>
 ```
 
+6. (Optional) By default, the default configuration doesn't include access to other services in the `Configuration` class.
+Should you require these, include the new class in your own `services.yml` to decorate the default service. If you use autowiring, you're done.
+Otherwise, configure the arguments of your class here as well.
+```yml
+services:
+    <your_class_here>:
+        decorates: kickin.exceptionhandler.configuration
+```
```

Comment:
We might actually consider autowiring the configuration service by default. This would mean removing the `EmptyConfiguration` class, but when you have one implemented, it should be available automatically by its interface. Or it just requires you a single line of configuration, while it is now three (see [here](https://symfony.com/doc/3.3/service_container/autowiring.html#working-with-interfaces)).

I'm not yet sure what this means for the configuration for the exception handler itself. If that doesn't work, this comment can be ignored :)

Reply:
autowiring works fine for typehinted services, but not for parameters. I can add that to the configuration and change the explanation here.